lutris/gui/application.py

```diff
@@ -697,22 +696,89 @@ def print_wine_runners(self):
 
     def install_runner(self, runner):
         if runner.startswith("lutris"):
-            Runner.prepare_wine_runner_cli(runner)
+            self.install_wine_cli(runner)
         else:
-            Runner.prepare_runner_cli(runner)
+            self.install_cli(runner)
 
     def uninstall_runner(self, runner):
         if "wine" in runner:
             print("Are sure you want to delete Wine and all of the installed runners?[Y/N]")
             ans = input()
             if ans.lower() in ("y", "yes"):
-                Runner.uninstall_runner_cli(runner)
+                self.uninstall_runner_cli(runner)
             else:
                 print("Not Removing Wine")
         elif runner.startswith("lutris"):
-            Runner.wine_runner_uninstall(runner)
+            self.wine_runner_uninstall(runner)
         else:
-            Runner.uninstall_runner_cli(runner)
+            self.uninstall_runner_cli(runner)
+
+    def install_wine_cli(self, version):
+        """
+        Downloads wine runner using lutris -r <runner>
+        """
+        runner = import_runner("wine")
+
+        WINE_DIR = os.path.join(settings.RUNNER_DIR, "wine")
+        runner_path = os.path.join(WINE_DIR, f"{version}{'' if '-x86_64' in version else '-x86_64'}")
+        if os.path.isdir(runner_path):
+            print(f"Wine version '{version}' is already installed.")
+        else:
+            from lutris.gui.dialogs import ErrorDialog
+            from lutris.gui.dialogs.download import simple_downloader
+            try:
+                runner().install(downloader=simple_downloader, version=version)
+                print(f"Wine version '{version}' has been installed.")
+            except RunnerInstallationError as ex:
+                ErrorDialog(ex.message)
+
+    def wine_runner_uninstall(self, version):
+        version = f"{version}{'' if '-x86_64' in version else '-x86_64'}"
+        WINE_DIR = os.path.join(settings.RUNNER_DIR, "wine")
+        runner_path = os.path.join(WINE_DIR, version)
+        if os.path.isdir(runner_path):
+            system.remove_folder(runner_path)
+            print(f"Wine version '{version}' has been removed.")
+        else:
+            print(f"Specified version of Wine is not installed: {version}. Please check if the Wine Runner and specified version are installed (--list-wine-runners can be used for that), and that the version specified is in the correct format.")
+
+    def install_cli(self, runner_name):
+        """
+        install the runner provided in prepare_runner_cli()
+        """
+
+        runner = import_runner(runner_name)
+        runner_path = os.path.join(settings.RUNNER_DIR, runner_name)
```

Comment:
This should probably be wrapped in try-except, like you did in uninstall_runner_cli bellow.